### PR TITLE
add value in Helm chant for log-level for kube-secrets-init 

### PR DESCRIPTION
**Pull request by Kydybets**
issue #151

### CODE REVIEW

**Comment by Kydybets** [comment]
@sagikazarmark  could you please review and provide your feedback?

**Review by sagikazarmark — CHANGES_REQUESTED** [review]
Thanks @Kydybets for the contribution. Please read the contribution guide and apply the steps there, particularly:

- generating docs
- bumping chart version
- updating change log

Thanks!

**Comment by Kydybets** [comment]
Thank you @sagikazarmark  for the comments.
I am new to contributing to open-source projects.
Unfortunately didn't find guide for contributors in the repo, try to do the same way like you do before for this chart

File: charts/kube-secrets-init/values.yaml [hunk 1]
```diff
@@ -15,6 +15,10 @@ image:
   # -- Image tag override for the default value (chart appVersion).
   tag: ""
 
+# define log-level argument for kube-secrets-init container:
+# could be "info", "debug", "warning", "error", "fatal", "panic"
```
[the remark this answers is not in the document]
Comment (Kydybets) [reply charts/kube-secrets-init/values.yaml]: commited

**Comment by Kydybets** [comment]
@sagikazarmark  is there any required?

**Comment by Kydybets** [comment]
@sagikazarmark  can you provide feedback please?

**Comment by sagikazarmark** [comment]
Can you please squash your commits to retrigger CI?

**Pull request closed without merging**